### PR TITLE
changes for Owasp security checks 

### PR DESCRIPTION
**Pull request by kalyan-chokkam**
- added fixes for security checks

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -26,8 +26,12 @@ android {
     }
 
     buildTypes {
+        debug {
+            minifyEnabled true
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
```
Comment (drtechie) [inline app/build.gradle:31]: Do we need progaurd for debug builds?

**Comment by drtechie** [comment]
@kalyan-chokkam looks fine.
Can you please add some documentation around setting the build config? For example, I didn't follow where
BuildConfig.ABHA_CLIENT_ID comes from. Thanks.

File: app/build.gradle (hunk 1 above)
Comment (kalyan-chokkam) [reply app/build.gradle]: removed

**Comment by kalyan-chokkam** [comment]
> @kalyan-chokkam looks fine. Can you please add some documentation around setting the build config? For example, I didn't follow where BuildConfig.ABHA_CLIENT_ID comes from. Thanks.

added secrets.properties file to add these properties, we can update the values in the file

**Review by drtechie — APPROVED** [review]

**Pull request merged**